core/src/main/java/org/elasticsearch/action/admin/cluster/allocation/ClusterAllocationExplanation.java

```diff
@@ -141,6 +150,11 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             // If we have unassigned info, show that
             if (unassignedInfo != null) {
                 unassignedInfo.toXContent(builder, params);
+                long delay = unassignedInfo.getLastComputedLeftDelayNanos();
+                builder.field("allocation_delay", TimeValue.timeValueNanos(delay));
+                builder.field("allocation_delay_ms", TimeValue.timeValueNanos(delay).millis());
+                builder.field("remaining_delay", TimeValue.timeValueNanos(remainingDelayNanos));
+                builder.field("remaining_delay_ms", TimeValue.timeValueNanos(remainingDelayNanos).millis());
```

Comment:
The reason I'm confused is, `allocation_delay` is set from a call to `unassignedInfo#getLastComputedLeftDelayNanos()`, and in `UnassignedInfo#updateDelay`, there is this code:

```
public long updateDelay(long nanoTimeNow, Settings settings, Settings indexSettings) {
        final long newComputedLeftDelayNanos = getRemainingDelay(nanoTimeNow, settings, indexSettings);
        lastComputedLeftDelayNanos = newComputedLeftDelayNanos;
        return newComputedLeftDelayNanos;
 }
```
so it seems the `lastComputedLeftDelayNanos` gets updated to the remaining delay?